x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/edit_policy.js

```diff
@@ -63,15 +63,30 @@ class EditPolicyUi extends Component {
       setSelectedPolicy(selectedPolicy);
     }
   };
+  getPolicyNameFromUri() {
```

Comment:
Can we extract this function into a service, and change it to accept URL as an argument? Then we can add tests to it, too.

Reply:
I actually went back to using a path param.  The double encoding fixed the mangling issue I was seeing.  There is far less code (this method is gone) and it is way easier to understand.

Reply:
Removed this code in favor of path params.